### ISSUE TITLE
Feature request: config option to disable pieces of RFTools

### ISSUE THREAD

**Issue by josephcsible** [McJty/RFTools#1338]
Can config options be added to disable individual pieces of RFTools, not registering the items or mentioning them in the in-game manual? For example:
* A "teleportation" option that would disable the matter receiver and all other teleporting-related blocks and items
* An "endergenic" option that would disable the endergenic generator, pearl injector, and ender monitor
* Individual options to disable more-or-less standalone things like the coal generator and elevator

**Comment by operinko** [comment]
This could be achieved with MineTweaker, by just removing the recipes.

**Comment by josephcsible** [comment]
True, but then they'd still be referenced in the manual and would still exist in the Forge registries
